src/test/java/io/wisetime/connector/jira/JiraConnectorPostTimeTest.java

```diff
@@ -286,7 +289,8 @@ void postTime_with_valid_group_should_succeed() {
     List<Long> updatedIssueTimes = timeSpentUpdateIssueCaptor.getAllValues();
     assertThat(updatedIssueTimes)
         .containsExactly(issue1.getTimeSpent() + 250, issue2.getTimeSpent() + 250)
-        .as("Time spent of both matching issues should be updated with new duration");
+        .as("Time spent of both matching issues should be updated with new duration. The duration should be " +
+            "split among the three tags even if one of them was not found.");
```

Comment:
Rationale: the third tag might be handled by another webhook. A third of the time will then be posted via the latter.

Reply:
FYI: In wt-agent implementation duration was divided only between matched tags:
E.g. if user posted time with tags [WT, WT-5163] with total duration 10minutes -> 10 minutes will be logged to ticket WT-5163. Source: https://stash.practiceinsight.io/stash/projects/WT/repos/wt-integrate-agents/browse/wt-agent-core/src/main/java/io/wisetime/agent/core/post_time/PersistPostedTimeSegment.java#157,237,268
line 237 - we filter tag list by isHandledByIntegration()
line 268 - invoke createTimeRowList()
line 157 - divide total duration between result tag list from line 237

Reply:
Thanks for checking @sukolenvo. The above implementation assumed that there would only be one agent per team. This logic will allow people to post more time than they have worked. Let's switch to the new logic of ignoring tags that the connector doesn't know about, while splitting the duration among all tags.